README.md

```diff
@@ -113,6 +113,14 @@ After building the library, you can install it using:
 cmake --install . --prefix ../dist
 ```
 
+When building and installing in a conda environment, you can specify the conda environment's prefix location:
+
+```bash
+cmake --install . --prefix $CONDA_PREFIX
+```
+
+Or alternatively configure this by passing `-DCMAKE_INSTALL_PREFIX=$CONDA_PREFIX` when invoking cmake to configure the project.
```

Comment:
Not sure if it is needed to mention both .. I have always seen `-DCMAKE_INSTALL_PREFIX=$CONDA_PREFIX` used (well, just in arrow), but the explanation here was already using `--prefix`